.travis.yml

```diff
@@ -3,15 +3,21 @@ language: c
 
 addons:
   apt:
+    sources:
+      - sourceline: 'deb http://us.archive.ubuntu.com/ubuntu/ trusty universe'
     packages:
       - vera++
-      #- gcc-arm-none-eabi
-      #- libnewlib-arm-none-eabi
+      - gcc-arm-none-eabi
+      - libnewlib-arm-none-eabi
```

Comment:
Do you really need newlib?

Reply:
Alas, yes. Otherwise the build blows up over trickery with chained including of `<stdint.h>`.